locale/it/LC_MESSAGES/django.po

```diff
@@ -7605,7 +7605,7 @@ msgid "Continue using Firefox as you normally would. Depending on how you use th
 msgstr "Potrai continuare a utilizzare Firefox come al solito. A seconda delle tue abitudini di navigazione, potresti addirittura non notare differenze."
 
 msgid "After a few weeks, the study will end on its own. Your Firefox will go back to normal."
-msgstr "Dopo alcune settimane l’indagine terminerà automaticamente e Firefox riprenderà ad essere quello di sempre.\n"
+msgstr "Dopo alcune settimane l’indagine terminerà automaticamente e Firefox riprenderà ad essere quello di sempre."
```

Comment:
Not really, I was under the impression I had it in a different PR already… but apparently not… might get re-done in a separate PR later.